docs/impeller/Flutter-GPU.md

```diff
@@ -1,49 +1,36 @@
+There is also an experimental 3D rendering package powered by Flutter GPU called [Flutter Scene](https://pub.dev/packages/flutter_scene).
 
-## Try out Flutter GPU
-
-Once released, Flutter GPU will be shipped as part of the Flutter SDK in the form of a Dart package called `flutter_gpu`. An early implementation of the `flutter_gpu` package is being developed under the [`lib/gpu` directory](https://github.com/flutter/engine/tree/main/lib/gpu) of the Flutter Engine repository.
-
-> [!CAUTION]
-> _All_ aspects of Flutter GPU are subject to breakage or removal at any time without prior deprecation notice or viable feature replacement. DO NOT rely on Flutter GPU for production projects at this time, but DO have fun playing with it and sharing your experiments with the community.
-
-Flutter GPU is currently unfinished, extremely experimental, and not well documented. [bdero](https://github.com/bdero) is actively developing and testing Flutter GPU against the MacOS desktop embedder; shader compilation and import likely don't function correctly on other platforms yet. However, if you wish to experiment with Flutter GPU, it is possible to do so without a custom Engine build:
-
-1. Update your Flutter checkout to the latest version in the [master channel](https://docs.flutter.dev/release/upgrade#other-channels).
-1. Clone [Flutter Engine](https://github.com/flutter/engine) and checkout the Engine commit that the Flutter master channel is currently pinned to. This can be found in the [`bin/internal/engine.version` file](https://github.com/flutter/flutter/blob/main/bin/internal/engine.version) of the main Flutter repository.
-    ```sh
-    git clone https://github.com/flutter/engine.git
-    cd engine
-    git reset --hard [PINNED_ENGINE_COMMIT]
-    ```
-1. Create a new Flutter project using the Flutter tool and add `flutter_gpu` as a dependency in `pubspec.yaml` with a local path pointing to the `lib/gpu` directory within the Flutter Engine repository cloned in step 2. For example:
-    ```yaml
-    dependencies:
-      flutter:
-        sdk: flutter
-      flutter_gpu:
-        path: ../engine/src/flutter/lib/gpu
-    ```
-1. From here, you can import the API and begin using it.
-    ```dart
-    import 'package:flutter_gpu/gpu.dart' as gpu;
-    ```
-    Check out this [examples repository](https://github.com/bdero/flutter-gpu-examples), which includes an example of [drawing a triangle](https://github.com/bdero/flutter-gpu-examples/blob/master/lib/triangle.dart), among other things.
+Flutter GPU is a low level API for building rendering packages from scratch. Graphics programming has a steep learning curve, and it's likely that most users will opt to use a higher level rendering package rather than build there own.
 
-## Reporting bugs
+## Distribution
+
+Flutter GPU can be used on the Flutter [master channel](https://docs.flutter.dev/release/upgrade#other-channels) by 
```

Comment:
Remove ` by` from the end of that line, and add a terminating `.`

Reply:
Done